### PR TITLE
Remove the `dev-` prefix from the names of all the generated images.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Nothing dev about them anymore.

https://trello.com/c/bfzgREhP/542-check-that-the-dev-part-in-the-names-of-mysql-postgres-broker-do-not-show-up-to-the-user

### CODE REVIEW

File: csm-extensions/services/dev-mssql/Makefile [hunk 1]
```diff
@@ -65,7 +65,7 @@ test:	test-format
 	go list ./... | grep -v vendor | go test -v)
 
 test-integration:
-	echo "$(WARN_CYN_COLOR)==>Use test-integration-manual target to run integration tests for dev-mssql $(NO_COLOR) ";n=0;\
+	echo "$(WARN_CYN_COLOR)==>Use test-integration-manual target to run integration tests for mssql $(NO_COLOR) ";n=0;\
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-mssql/Makefile]: Will check.

File: csm-extensions/services/dev-mysql/O [hunk 2]
```diff
@@ -0,0 +1 @@
+/fast/dev/scf-1/output
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-mysql/O]: This is a soft-link and definitely wrongly committed. Will squash. And rebase to expunge from history too.

File: csm-extensions/services/dev-postgres/O [hunk 3]
```diff
@@ -0,0 +1 @@
+/fast/dev/scf-1/output
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-postgres/O]: Ditto

File: csm-extensions/services/dev-mysql/O (hunk 2 above)
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-mysql/O]: Fixed with commit e8676fb35ec3c1f391036261001d50fbd7ca6b94, no rebase yet.

File: csm-extensions/services/dev-postgres/O (hunk 3 above)
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-postgres/O]: Fixed with commit e8676fb35ec3c1f391036261001d50fbd7ca6b94, no rebase yet.

**Comment by andreas-kupries** [comment]
Re: additional references - Apologies, bad me - I grepped and then apparently excluded too much, due to the mass of bogus positives in the go code files (All the imports). Time to grep again and be more careful ...

**Comment by mook-as** [comment]
No worries about the bad references… I had trouble skipping all the import references too. Hopefully we caught everything between the two of us.

**Comment by andreas-kupries** [comment]
Rebased. Kept it as 3 commits, for makefiles, charts, and the integration_test.go files.
Expunged only the soft links from the history.

Should be ok now.

File: csm-extensions/services/dev-mssql/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-mssql/Makefile]: Oops. Missed this one. Checking now ...

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply csm-extensions/services/dev-mssql/Makefile]: And done, commit 4359455e7fd0857e24112fbe94d79b69309746fa

**Pull request merged**